courses/machine_learning/asl2/recommendation_systems/als_bqml.ipynb

```diff
@@ -0,0 +1,1550 @@
+{
```

Comment:
<p>Add copyright block here.  See other notebooks for an example.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/517/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>